push-messaging-and-notifications/main.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+var API_KEY = 'AIzaSyBBh4ddPa96rQQNxqiq_qQj7sq1JdsNQUQ';
+
+var isPushEnabled = false;
+
+function sendToServer(subscriptionId, endpoint) {
+  // The curl command to trigger a push message straight from GCM
+  var curlCommand = 'curl --header "Authorization: key=' + API_KEY +
+    '" --header Content-Type:"application/json" ' + endpoint + 
+    ' -d "{\\"registration_ids\\":[\\"' + subscriptionId + '\\"]}"';
+
+  // Below is a curl command to test sending a push message
+  console.log(curlCommand);
```

Comment:
That part looks good (assuming the decision is that you can go ahead sharing the cURL command).

I think it would help to do something similar for all the other instances where you call `console` methods. Or, if you'd really prefer not to, at least give developers viewing the page a heads-up that there's going to be important info logged to the JS console.

There are a bunch of error messages and info messages that we wouldn't want developers to miss seeing.